lib/cocoapods.rb

```diff
@@ -45,6 +45,7 @@ def message
   autoload :ExternalSources,           'cocoapods/external_sources'
   autoload :Installer,                 'cocoapods/installer'
   autoload :HooksManager,              'cocoapods/hooks_manager'
+  autoload :Podfile,                   'cocoapods/podfile'
```

Comment:
this shouldnt be necessary, since we eagerly require Core

Reply:
even if this file is not part of Core?

Reply:
ah, in that case I dont think we should autoload it -- since that constant is _also_ defined in core

Reply:
ok, will update this to load only in the places that use it